src/Conversion/ONNXToTorch/NN/PaddingOp.cpp

```diff
@@ -2,7 +2,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
```

Comment:
@ramakota-helprack 

I'm a bit confused about the changes here.  They seem to delete the implementation that you made with the attributes.  Can you please check that your changes are there at the proper place?  If not, then please make necessary modifications.

Reply:
Resolved